contracts/0.8.25/vaults/StakingVault.sol

```diff
@@ -554,6 +646,44 @@ contract StakingVault is IStakingVault, OwnableUpgradeable {
      */
     event BeaconChainDepositsResumed();
 
+    /**
+     * @notice Emitted when ether is deposited to `DepositContract`.
+     * @param _sender Address that initiated the deposit.
+     * @param _deposits Number of validator deposits made.
+     * @param _totalAmount Total amount of ether deposited.
+     */
+    event DepositedToBeaconChain(address indexed _sender, uint256 _deposits, uint256 _totalAmount);
+
+    /**
+     * @notice Emitted when a validator exit request is made.
+     * @param _sender Address that requested the validator exit.
+     * @param _pubkeys Public key of the validator requested to exit.
+     * @dev    Signals `nodeOperator` to exit the validator.
+     */
+    event ValidatorExitRequested(address indexed _sender, bytes _pubkeys);
```

Comment:
```suggestion
    event ValidatorExitRequested(address _sender, bytes indexed _pubkey);
```

Reply:
indexed are actually stoked as keccak256 hash of the value, do we need that?